README.md

```diff
@@ -117,7 +117,7 @@ See the [changelog](https://github.com/ansible-collections/community.docker/tree
 
 This collection is primarily licensed and distributed as a whole under the GNU General Public License v3.0 or later.
 
-See [LICENSES/GPL-3.0-or-later.txt](https://github.com/ansible-collections/community.docker/blob/main/LICENSES/GPL-3.0-or-later.txt) for the full text.
+See [LICENSES/GPL-3.0-or-later.txt](https://github.com/ansible-collections/community.docker/blob/main/COPYING) for the full text.
```

Comment:
/me wishes Github would properly handle symlinks instead of creating useless other features and committing license violations en masse (Copilot), but this is probably the best solution.

Reply:
:+1: